code/controllers/subsystem/timer.dm

```diff
@@ -260,6 +260,12 @@ SUBSYSTEM_DEF(timer)
 	// Add all timed events from the secondary queue as well
 	alltimers += second_queue
 
+	for (var/datum/timedevent/t in alltimers)
```

Comment:
```suggestion
	for (var/datum/timedevent/t as anything in alltimers)
```

Reply:
What difference does it make?

Reply:
optimization because it means it doesn't typecheck in a list where it knows everything is a /datum/timedevent anyways

Reply:
it needs to typecheck thou

this is recovery code that can't assume everything is proper.

Reply:
actually, nothing in this proc does that, so this can skip it too.

@Bizzonium it makes it much much faster as it keeps byond from having to check every timer in the loop for properness (is_type(x)) at runtime is expensive in some cases for reasons that don't make sense.